continuum/scenarios/instance_incremental.py

```diff
@@ -34,34 +34,15 @@ def __init__(
     def _setup(self, nb_tasks: int) -> int:
         x, y, t = self.cl_dataset.get_data()
 
-        if t is None and nb_tasks <= 0:
-            raise ValueError(f"You need to specify a number of tasks > 0, not {nb_tasks}.")
-        if t is None:  # If the dataset didn't provide default task ids:
+        if nb_tasks is not None and nb_tasks > 0:  # If the user wants a particular nb of tasks
```

Comment:
t is none case has been deleted here.

Reply:
Yes, it's on purpose.

If t is none, then the user MUST specify a nb_tasks, otherwise that's an error

Reply:
Note also that if the user had specify a nb_tasks, it overrides the default number of tasks of the dataset.

Reply:
ok good